files/zh-cn/learn/common_questions/design_and_accessibility/common_web_layouts/index.md

```diff
@@ -3,6 +3,8 @@ title: 常见 web 布局都包含什么？
 slug: Learn/Common_questions/Design_and_accessibility/Common_web_layouts
 ---
 
+{{QuicklinksWithSubPages("/zh-CN/docs/Learn/Common_questions")}}
```

Comment:
is there any pages have using the full path?

For this page, https://github.com/mdn/content/blob/f22e72998f4e8f48b18ef358521bfc9ad1ae9446/files/en-us/learn/common_questions/design_and_accessibility/common_web_layouts/index.md?plain=1#L7

Reply:
hmmm, no

but currently only if full path is used can the sidebar display properly 🤔

Reply:
I'm not sure if this will cause a missing content issue, when there are documents under this tree that aren't localized.